nix_mx.cc

```diff
@@ -184,7 +184,7 @@ void mexFunction(int            nlhs,
             .reg("create_range_dimension", &nix::DataArray::createRangeDimension)
             .reg("create_alias_range_dimension", &nix::DataArray::createAliasRangeDimension)
             .reg("create_sampled_dimension", &nix::DataArray::createSampledDimension);
-        methods->add("DataArray::delete_dimension", nixdataarray::delete_dimension);
+		methods->add("DataArray::delete_dimensions", nixdataarray::delete_dimensions);
```

Comment:
must have happened when my ms account got suspended and i had to set up vs anew. i'll change it.